### PR TITLE
feat: added Home screen 

### PR DESCRIPTION
**Pull request by md-amir-gauhar**
Screenshot (212)
Screenshot (214)

### CODE REVIEW

**Review by Hamza-777 — APPROVED** [review]
Good ui and code, looks good to merge

File: src/pages/Home.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import { useData } from '../context/DataContext'
+
+import '../styles/Home.css'
+
+const Home = () => {
+  const { videos, categories, sortByCategory, dispatch } = useData()
+
+  const clickHandler = (categoryName) => {
+    dispatch({
+      type: "SORT_BY_CATEGORY",
```
Comment (Pratik1005) [inline src/pages/Home.js:12]: use constants for action types to avoid typos

File: src/pages/Home.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      <div className="categories-container">
+        {
+          categories.map(({ _id, categoryName, active }) => (
+            <span key={_id} className={`category ${active && "is-active"}`} onClick={() => clickHandler(categoryName)}>{categoryName}</span>
+          ))
+        }
+        {/* <span>All</span>
+        <span>AMV</span>
+        <span>Anime Trailer</span>
+        <span>Anime Opening</span> */}
```
Comment (Pratik1005) [inline src/pages/Home.js]: remove commented code if not needed

**Review by Pratik1005 — APPROVED** [review]
Good UI

**Pull request merged**